Problem2.py

```diff
@@ -0,0 +1,48 @@
+// Time Complexity : O(1) ?
+// Space Complexity : O(1)
```

Comment:
Time complexity is correct. Please check the Space Complexity.  2 stacks are being used here.